man/installspec.Rd

```diff
@@ -4,12 +4,21 @@
 \alias{installspec}
 \title{Install the kernelspec to tell Jupyter (or IPython ≥ 3) about IRkernel.}
 \usage{
-installspec(user = TRUE)
+installspec(user = TRUE, name = "ir", displayname = "R")
```

Comment:
I couldn't get `devtools::document()` to change this to single quotes...

http://r-pkgs.had.co.nz/man.html